sync2/rangesync/wire_conduit.go

```diff
@@ -58,7 +74,7 @@ var _ Conduit = &wireConduit{}
 func startWireConduit(ctx context.Context, s io.ReadWriter, opts ...ConduitOption) *wireConduit {
 	c := &wireConduit{
 		stream: s,
-		sendCh: make(chan SyncMessage, writeQueueSize),
+		sendCh: make(chan SyncMessage, sendQueueSize),
```

Comment:
buffered channel with 200k potential entries..? usually the golang styleguide says buffered channel should be `one or none`

Reply:
There's a TODO comment explaining motivation for this channel size above:
https://github.com/spacemeshos/go-spacemesh/pull/6404/files#diff-a3083340af74afba860bcaafecd95af73e3982f237ca0620a41890eb744ad28bR16-R31
This big channel is not good indeed, problem is that it's not exactly easy to fix.
Still, I'll try to get rid of this huge channel in a followup PR before we finalize syncv2.